wire-ios-request-strategy/Sources/Request Strategies/Feature configurations/Payload/FeatureConfigsPayload.swift

```diff
@@ -50,6 +50,7 @@ struct FeatureConfigsPayloadAPIV6: Decodable {
     let channels: FeatureStatusWithConfig<Feature.Channels.Config>?
     let consumableNotifications: FeatureStatus?
     let chatBubbles: FeatureStatus?
+    let assetAuditLog: FeatureStatus?
```

Comment:
question: do we have to implement it here? As I understood it would be sufficient to implement it only in WireNetwork/ WireDomain. We should stop adding new fields in legacy.

It would make sense to remove the usage of FeatureConfig

Reply:
ok but it can't be a requirement, no? you would not expect even if running old sync recieve this featureflag unless backend backports it, which I doubt they do

Reply:
done